### PR TITLE
[AAE-14708] Fix datetime values not working properly

### PR DESCRIPTION
**Pull request by DudaRobert**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-14708

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/src/lib/pipes/moment-datetime-utc-to-local.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import moment, { Moment } from 'moment';
+
+@Pipe({ name: 'adfMomentDateTimeUtcToLocal' })
+export class MomentDateTimeUtcToLocalPipe implements PipeTransform {
+    transform(value: moment.MomentInput): Moment | null {
+        if(value) {
```
[the remark this answers is not in the document]
Comment (DudaRobert) [reply lib/core/src/lib/pipes/moment-datetime-utc-to-local.pipe.ts:24]: agreed, going to do the same in https://github.com/Alfresco/alfresco-ng2-components/pull/8606/files#diff-ac8476e972ca53928b62d86036ff30c65203f7864a48479cc60cd7ae0fd19396L85

File: lib/core/src/lib/pipes/moment-datetime-utc-to-local.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+import moment, { Moment } from 'moment';
+
+@Pipe({ name: 'adfMomentDateTimeUtcToLocal' })
```
Comment (DenysVuika) [inline lib/core/src/lib/pipes/moment-datetime-utc-to-local.pipe.ts:21]: we are moving away from the moment, maybe you could use the https://date-fns.org/ instead? it is already used by the ADF components

**Comment by arditdomi** [comment]
@DudaRobert  what is the status of this? I am going to add the next-version as is listed a Cat 3, but let me know if for some reason is needed for Invader

**Comment by DudaRobert** [comment]
> @DudaRobert what is the status of this? I am going to add the next-version as is listed a Cat 3, but let me know if for some reason is needed for Invader

This is a little embarassing, but I completely forgot about this issue 🙈.

**Pull request closed without merging**